tools/df/docker/bird-fi-test/run.sh

```diff
@@ -0,0 +1,10 @@
+IMAGE=bird-fi-test
```

Comment:
I'm not sure where this script would be ran. It certainly could not be ran in our current docker environment in ECS. If you want to be testing bird I can think of two ways to do it, but it depends on if you're trying to test our custom Bird or the Deepfield code that utilizes it. If you're trying to test the custom Bird debian I recommend running it from our current docker container (pipedream-dev). This seems like the easiest way to me, as many of our other fi tests do this with various Deepfield daemons and other services, using environment setup/teardown steps to start and stop daemons for the corresponding test suite. 

An alternative is going down the route of a new container (the bird-fi-test container). This would be similar to how we have containers for dfzook/kudu/impala/mysql. While it is more aligned with the "right" way of doing things with docker, it is it's a poor way to test the services running in these containers, as you cannot get the stdout from linked containers in ECS as a part of the test result in Jenkins. This method would be great if you're only trying to test Deepfield code that interacts with Bird.

Are you trying to test the custom bird build? or the code we use to interact with it?